src/h3lib/lib/h3Index.c

```diff
@@ -860,6 +885,29 @@ void H3_EXPORT(h3GetFaces)(H3Index h3, int* out) {
     }
 }
 
+/**
+ * pentagonIndexCount returns the number of pentagons (same at any resolution)
+ *
+ * @return int count of pentagon indexes
+ */
+int H3_EXPORT(pentagonIndexCount)() { return 12; }
```

Comment:
Nit: Move `#define NUM_PENTAGONS 12` from `generateNumHexagons.c` to `constants.h` and reuse it here?